pom.xml

```diff
@@ -215,6 +218,71 @@
                         </execution>
                     </executions>
                 </plugin>
+                <plugin>
+                    <artifactId>maven-dependency-plugin</artifactId>
+                    <version>2.10</version>
+                    <executions>
+                        <execution>
+                            <id>unpack</id>
+                            <phase>generate-sources</phase>
+                            <goals>
+                                <goal>unpack</goal>
+                            </goals>
+                            <configuration>
+                                <artifactItems>
+                                    <artifactItem>
+                                        <!-- includes common copyright headers for sources -->
+                                        <groupId>org.kaazing</groupId>
+                                        <artifactId>community.license</artifactId>
+                                        <type>jar</type>
+                                    </artifactItem>
+                                </artifactItems>
+                                <overWriteReleases>true</overWriteReleases>
+                                <overWriteSnapshots>true</overWriteSnapshots>
+                                <failOnMissingClassifierArtifact>true</failOnMissingClassifierArtifact>
+                            </configuration>
+                        </execution>
+                    </executions>
+                </plugin>
+                <plugin>
+                    <groupId>com.mycila</groupId>
+                    <artifactId>license-maven-plugin</artifactId>
+                    <version>2.3</version>
+                    <configuration>
+                        <header>${project.build.directory}/dependency/COPYRIGHT-community</header>
+                        <failIfMissing>true</failIfMissing>
+                        <useDefaultExcludes>false</useDefaultExcludes>
+                        <includes>
+                            <include>src/**</include>
+                        </includes>
+                        <excludes>
+                            <exclude>**/*.png</exclude>
```

Comment:
You can exclude files, but they would have to match some sort of regexp, I believe.  And since our files that don't match have no extension, there isn't really a good regexp to match to.  (*.ServiceFactorySpi doesn't seem right.)
  If you wanted to do it per project configuration, then you could list the full filename.  It seems overkill to me, as we have so much output during our builds that a few warnings about "file extension unknown" doesn't seem like a big deal to me.  But, this back-and-forth here is in the middle of an "excludes" configuration.  Just exclude whatever full filename you would want to exclude.